x/dex/types/keys.go

```diff
@@ -207,3 +207,6 @@ func JITGoodTilTime() time.Time {
 const (
 	ExpiringLimitOrderGas = 10_000
 )
+
+// Dummy Address used for simulate queries
+const DummyAddress = "neutron1pq7j6za5zjcl3um9t5gfyleues336tv04tyq0k"
```

Comment:
We do not simulate bank operation during estimate queries, do we still need a dummy address?

Reply:
Still required for message validation for MHS, deposit and place limit order. 
We could have a separate validation function for the simulate queries. But I prefer to re-use as much logic as possible.

Reply:
Validation only? Then let me rephrase the question, why do we want to fill the fields on behalf of the user? It's a simulation query and i assume user take care about address fields.